OMCompiler/Compiler/Util/Flags.mo

```diff
@@ -1355,7 +1355,12 @@ constant ConfigFlag FLAT_MODELICA = CONFIG_FLAG(141, "flatModelica",
   Gettext.gettext("Outputs experimental flat Modelica."));
 constant ConfigFlag FMI_FILTER = CONFIG_FLAG(142, "fmiFilter", NONE(), EXTERNAL(),
   ENUM_FLAG(FMI_INTERNAL, {("none", FMI_NONE), ("internal", FMI_INTERNAL), ("protected", FMI_PROTECTED), ("blackBox", FMI_BLACKBOX)}),
-  SOME(STRING_OPTION({"none", "internal", "protected", "blackBox"})),
+  SOME(STRING_DESC_OPTION({
+    ("none", Gettext.gettext("All variables will be exposed, even variables that are introduced by the symbolic transformations. Hence, this is intended to be used for debugging.")),
+    ("internal", Gettext.gettext("All internal variables introduced by the symbolic transformations are filtered out. Only the variables from the actual Modelica model are exposed (with minor exceptions, e.g. for state sets).")),
```

Comment:
"internal"  to  Gettext.gettext("Default Option, All internal variables introduced by the symbolic transformations are filtered out. Only the variables from the actual Modelica model are exposed (with minor exceptions, e.g. for state sets)."))

Reply:
Not sure if its a good idea to write this in the text. Users guide automatically read it from the default option from the default argument of flag. https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/omchelptext.html#omcflag-fmifilter

Reply:
Then it is fine